versions.tf

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
We don't need this file. It is being produced by the code migration utility.

Reply:
It's limiting to a specific Terraform version so I think it's useful

Reply:
If this is not good practice to keep this files for modules then I will remove it